codeforboston/maple
# faq page functionality

# Summary
#1271

figma:
https://www.figma.com/file/pGGIX2tmCWJNGU1sAYQ91v/FAQ?type=design&node-id=4001-29831&mode=design&t=5BYPwvKnYrgb4LUY-0

ABOUT page that includes collapse functionality - click on questions to see answers displayed 

Json to store Faq content for parsing - md docs limited parsing

Accordion did not provide flexibility for needed modifications- used Bootstrap Collapse 

Dropdown feature is future version and is not included

# Checklist


# Screenshots

https://github.com/codeforboston/maple/assets/101157878/1924caa1-356c-4555-b00e-8f6e2d9d0dd5

# Known issues
n/a

# Steps to test/reproduce

1. Go to Faq page
1. Click on questions and see collapse feature
